src/link.js

```diff
@@ -1,24 +1,49 @@
 var React = require('react');
 
 module.exports = React.createClass({
-  render: function () {
+  contextTypes: {
+    controller: React.PropTypes.object
+  },
 
-    if (typeof this.props.signal !== 'function') {
-      throw new Error('Cerebral React - You have to pass a signal to the LINK component');
-    }
+  componentDidMount() {
+
+    if (typeof this.props.signal === 'function') {
+      this.signal = this.props.signal;
+    } else if (typeof this.props.signal === 'string') {
+      var signalPath = this.props.signal.split('.');
+      var signalParent = this.context.controller.signals;
 
-    if (typeof this.props.signal.getUrl !== 'function') {
-      throw new Error('Cerebral React - The signal passed is not bound to a route');
+      while(signalPath.length - 1) {
+        signalParent = signalParent[signalPath.shift()];
+      }
+      this.signal = signalParent[signalPath];
```

Comment:
We should probably check if the path actually exists. In case you type something wrong :-)